src/inc/regdisp.h

```diff
@@ -406,6 +406,13 @@ inline void FillRegDisplay(const PREGDISPLAY pRD, PT_CONTEXT pctx, PT_CONTEXT pC
     FillContextPointers(&pRD->ctxPtrsOne, pctx);
 
 #if defined(_TARGET_ARM_)
+    // Fill volatile context pointers. They can be used by GC in the case of the leaf frame
```

Comment:
Should we add the arm64 volatile context pointers setting too like we do in the master branch?

Reply:
This is a hotfix for a customer attempting to debug on a DragonBoard.  UWP6.2 is based off of the .NET Core 2.1 codebase so I assume there are many other fixes we would need to support ARM64.  Since this is the UWP 6.2 CoreCLR branch, the only supported platform is Windows 10 and ARM32, X86, AMD64 architectures.